plottr/plot/pyqtgraph/autoplot.py

```diff
@@ -212,11 +219,22 @@ def _1dPlot(self, plotItem: PlotItem) -> None:
             return subPlot.plot.plot(x.flatten(), y.flatten(), name=name,
                                      pen=mkPen(color, width=1), symbol=symbol, symbolBrush=color,
                                      symbolPen=None, symbolSize=symbolSize)
-        else:
+        elif plotItem.plotDataType == PlotDataType.log10_line1d:
+            name = plotItem.labels[-1] if isinstance(plotItem.labels, list) else ''
```

Comment:
This syntax is cool, but it makes it harder to read compared to a normal if else block